src/account/BiconomySmartAccountV2.ts

```diff
@@ -442,6 +442,49 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return result
   }
 
+  /**
+   * Returns a single token balance for the smartAccount instance.
+   *
+   * This method will fetch only the token balance of a given token address for the smart account
+   *
+   * @param address - Required. Address of the asset to fetch the balance of. If not passed, the native balance of the smart account will be fetched.
+   * @returns Promise<bigint> - The token balance of the smart account
+   *
+   * @example
+   * import { createClient } from "viem"
+   * import { createSmartAccountClient } from "@biconomy/account"
+   * import { createWalletClient, http } from "viem";
+   * import { polygonAmoy } from "viem/chains";
+   *
+   * const signer = createWalletClient({
+   *   account,
+   *   chain: polygonAmoy,
+   *   transport: http(),
+   * });
+   *
+   * const token = "0x747A4168DB14F57871fa8cda8B5455D8C2a8e90a";
+   * const smartAccount = await createSmartAccountClient({ signer, bundlerUrl });
+   * const tokenBalance = await smartAccount.getTokenBalance(token);
+   *
+   * console.log(tokenBalance);
+   * // 83546776652078153068n
+   *
+   */
+  public async getTokenBalance(address?: Hex): Promise<bigint> {
```

Comment:
this function returns a users ethBalance, so the name is now less relevant. `getBalance` seems more appropriate